addon/styles/partials/_uni-select.scss

```diff
@@ -1,3 +1,29 @@
+.uni-select-wrapper--use-alias {
+  position: relative;
+
+  select.uni-select {
+    z-index: 2;
+    opacity: 0;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+
+    &:focus + div.uni-select {
+        border: 2px solid color('input', 'border-focus');
+    }
+  }
+
+  div.uni-select {
+    z-index: 1;
+    @include align-items-center();
```

Comment:
hoist @includes to top of class, also div.uni-select is an overqualification, is it really needed?